Source/pages/checkin.php

```diff
@@ -79,7 +82,7 @@
 $t_vcs = SourceVCS::repo( $t_repo );
 
 # Let the plugins handle commit data
-$t_changesets = $t_vcs->commit( $t_repo, $f_data );
+$t_changesets = $t_vcs->commit( $t_repo, $f_data, $f_revprop );
```

Comment:
Will cause issues with VCS plugins that do not have this extra parameter in commit() method definition.